src/lib/requests/baseRequests.js

```diff
@@ -172,6 +172,49 @@ class BaseRequests {
                 throw e;
             });
     }
+    
+    /**
+     * @function _patch
+     * @description
+     *  Perform a HTTP PATCH request.
+     *
+     * @param {string} url - The url of the resource
+     * @param {string} resourceType - The 'type' of resource, as defined in the Mojaloop specification
+     * @param {string | undefined} dest - The destination participant. Leave empty if participant is unknown (e.g. `GET /parties`)
+     * @param {Object} headers - Optional additional headers
+     * @param {*} query - Optional query parameters
+     * @param {*} responseType - Optional, defaults to `Mojaloop`
+     */
+    async _patch(url, resourceType, body, dest, headers = {}, query = {}, responseType = ResponseType.Mojaloop) {
```

Comment:
there is a need for unit tests

Reply:
I can't seem to find unit testing for the other BaseRequests

Reply:
so you need to be creative :-D

Reply:
but this method is tested explicitly in thirdpartyRequests because there is a mock of http.__write & http.__request which is used by request.js so no need to test -> but please, do a double check in code coverage report are all paths for `patch` tested